### ISSUE TITLE
DatePicker: High contrast issue with today's date

### ISSUE THREAD

**Issue by lrabinowicz** [OfficeDev/office-ui-fabric-react#11617]
### Environment Information

- **Package version(s)**:  office-ui-fabric-react@7.78.0
- **Browser and OS versions**: Microsoft Chromium Edge, Windows 10

### Describe the issue
When high contrast mode is on, you can't see today's date in the datePicker

#### Actual behavior:

Repro steps:

1- Turn on high contrast mode
2- Open Chromium Edge 
3- Go to fabric UI and search for the date picker (https://developer.microsoft.com/en-us/fabric#/controls/web/datepicker)
4- Open any datePicker (for example default datePicker)

![TodaysDate-DatePickerBug](https://user-images.githubusercontent.com/12565958/71859925-8e199600-30a5-11ea-9f15-bf8cb492dec5.png)
#### Expected behavior:

You should be able to see all the dates of the month.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

**Comment by khmakoto** [comment]
Hi @lrabinowicz, this is actually an issue with `Calendar` itself, not only with `DatePicker`. I'll have a fix for this shortly.
